packages/linux-kernel/arch/arm64/configs/vyos_defconfig

```diff
@@ -2216,312 +2362,201 @@ CONFIG_DM_DELAY=m
 # end of IEEE 1394 (FireWire) support
 
 CONFIG_NETDEVICES=y
-CONFIG_MII=y
+CONFIG_MII=m
 CONFIG_NET_CORE=y
-CONFIG_BONDING=m
+# CONFIG_BONDING is not set
 CONFIG_DUMMY=m
 CONFIG_WIREGUARD=m
 # CONFIG_WIREGUARD_DEBUG is not set
 # CONFIG_EQUALIZER is not set
 # CONFIG_NET_FC is not set
-CONFIG_IFB=m
+# CONFIG_IFB is not set
 # CONFIG_NET_TEAM is not set
 CONFIG_MACVLAN=m
-# CONFIG_MACVTAP is not set
-CONFIG_IPVLAN_L3S=y
-CONFIG_IPVLAN=m
-# CONFIG_IPVTAP is not set
-CONFIG_VXLAN=m
+CONFIG_MACVTAP=m
+# CONFIG_IPVLAN is not set
+CONFIG_VXLAN=y
 # CONFIG_GENEVE is not set
 # CONFIG_BAREUDP is not set
 # CONFIG_GTP is not set
 # CONFIG_MACSEC is not set
-CONFIG_NETCONSOLE=m
-# CONFIG_NETCONSOLE_DYNAMIC is not set
-CONFIG_NETPOLL=y
-CONFIG_NET_POLL_CONTROLLER=y
-CONFIG_TUN=m
+# CONFIG_NETCONSOLE is not set
+CONFIG_TUN=y
+CONFIG_TAP=m
 # CONFIG_TUN_VNET_CROSS_LE is not set
 CONFIG_VETH=m
+CONFIG_VIRTIO_NET=y
```

Comment:
This needs to be a module (=m), otherwise `cat /sys/class/net/eth0/device/driver/module` fails inside https://github.com/vyos/vyos-1x/blob/02f2700d4e3742bacf534c5bd6c4118c66a18aff/python/vyos/ethtool.py#L59.

With the driver name not recognized it fails to realize that no flow control and negotiation is supported as specified in "_drivers_without_speed_duplex_flow". This completely breaks configuring the device.